tests/blockchain/test_blockchain.py

```diff
@@ -2589,3 +2589,17 @@ async def test_get_header_blocks_in_range_tx_filter(self, empty_blockchain):
             != blocks_without_filter[header_hash].transactions_filter
         )
         assert blocks_with_filter[header_hash].header_hash == blocks_without_filter[header_hash].header_hash
+
+    @pytest.mark.asyncio
+    async def test_get_blocks_at(self, empty_blockchain, default_1000_blocks):
+        b = empty_blockchain
```

Comment:
What was the reason for the introduction of the `b`?

Reply:
it looks like it may be to silence an unused-variable warning